source/_integrations/scrape.markdown

```diff
@@ -263,3 +263,46 @@ scrape:
 ```
 
 {% endraw %}
+
+### Container cleaning by CleanProfs in The Netherlands
+
+The example gets the container type and container cleaning date for the next two cleanings
+
+```yaml
+# Configuration entry. Change postal code and house number to your own address.
+scrape:
+  - resource: https://crm.cleanprofs.nl/search/planning
+    method: POST
+    payload: zipcode=5624JW&street_number=17
```

Comment:
Why are the resource duplicated? You can call a post to a single resource once, and add multiple sensors to that single resource.

This right now, posts 4 times for no reason.

../Frenck